components/CellIDishID.js

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import {
+  Image,
+  Platform,
+  StyleSheet,
+  Text,
+  TouchableOpacity,
+  View,
+  FlatList,
+  CheckBox,
+  TextInput,
+} from "react-native-web";
+import styled from "styled-components";
+import { ButtonPlus, ButtonMinus } from "./PlusMinusButton";
+
+export default function CellItemDetail({ item, navigation }) {
+  // console.log(item)
```

Comment:
just delete don't need useless comments in our codebase

Reply:
ok